.github/workflows/zulip-ci.yml

```diff
@@ -0,0 +1,157 @@
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  bionic_backend_frontend:
+    name: Ubuntu 18.04 Bionic (Python 3.6, backend + frontend)
+    runs-on: ubuntu-latest
+
+    # This docker image was created by a generated Dockerfile at:
+    #   tools/ci/images/bionic/Dockerfile
+    # Bionic ships with Python 3.6.
+    container: mepriyank/actions:bionic
+    env:
+      # GitHub Actions sets HOME to /github/home which causes
+      # problem later in provison and frontend test that runs
+      # tools/setup/postgrest-init-dev-db because of .pgpass.
+      # psql expects .pgpass to be at /home/github/.pgpass and
+      # setting home to `/home/github/` ensures it written there.
+      HOME: /home/github/
+
+    steps:
+    # Upgrade git before using checkout actions because it downloads
+    # the code using Rest API if git version is less than v2.18,
+    # which will cause errors in tests and provision.
+    - name: Upgrade git for bionic
+      run: |
+        sudo apt-get install -y software-properties-common
+        sudo add-apt-repository ppa:git-core/ppa -y
+        sudo apt-get update
+        sudo apt-get install -y git
+
+    - name: Add required permissions
+      run: |
+        # The checkout actions doesn't clone to ~/zulip or allow
+        # use to use the path option to clone outside the current
+        # /__w/zulip/zulip directory. Since this directory is owned
+        # by root we need to change it's ownership to allow the
+        # github user to clone the code here.
+        # Note: /__w/ is a docker volume mounted to $GITHUB_WORKSPACE:
+        #   /home/runner/work/
+        # that GitHub Actions want use to clone the code to.
+        sudo chown -R github .
+
+        # This is the GitHub actions specific cache directory the
+        # the current github user must be able to access for the
+        # cache action to work.
+        sudo chmod -R 0777 /__w/_temp/
+
+    - uses: actions/checkout@v2
+
+    - name: Create cache directories
+      run: |
+        dirs=(/srv/zulip-{npm,venv,emoji}-cache)
+        sudo mkdir -p "${dirs[@]}"
+        sudo chown -R github "${dirs[@]}"
+
+    - name: Restore node_modules cache
+      uses: actions/cache@v2
+      with:
+       path: /srv/zulip-npm-cache
+       key: v1-yarn-deps-${{ github.job }}-${{ hashFiles('package.json') }}-${{ hashFiles('yarn.lock') }}
+       restore-keys: v1-yarn-deps--${{ github.job }}
+
+    - name: Restore python cache
+      uses: actions/cache@v2
+      with:
+        path: /srv/zulip-venv-cache
+        key: v1-venv-${{ github.job }}-${{ hashFiles('requirements/thumbor-dev.txt') }}-${{ hashFiles('requirements/dev.txt') }}
+        restore-keys: v1-venv-${{ github.job }}
+
+    - name: Restore emoji cache
+      uses: actions/cache@v2
+      with:
+        path: /srv/zulip-emoji-cache
+        key: v1-emoji-${{ github.job }}-${{ hashFiles('requirements/thumbor-dev.txt') }}-${{ hashFiles('requirements/dev.txt') }}
```

Comment:
I think these are not the keys we need for the emoji cache. The one we use in Circle CI config should be here.

Reply:
Ahh, this is a copy-paste error which I think I missed because we `v1-venv` twice. Fixed, thanks.